benchmarks/eko/benchmark_evol_to_unity.py

```diff
@@ -10,25 +9,27 @@
 from eko.io import types
 from eko.io.runcards import OperatorCard, TheoryCard
 from eko.runner.legacy import Runner
+from eko.quantities.couplings import CouplingsInfo
 
 # from ekore.matching_conditions.operator_matrix_element import OperatorMatrixElement
 
 
 def update_cards(theory: TheoryCard, operator: OperatorCard):
-    theory.couplings = types.CouplingsRef(
-        alphas=types.FloatRef(value=0.35, scale=float(np.sqrt(2))),
-        alphaem=types.FloatRef(value=0.007496, scale=nan),
+    theory.couplings = CouplingsInfo(
+        alphas=0.35,
+        alphaem=0.007496,
+        scale=float(np.sqrt(2)),
         max_num_flavs=6,
         num_flavs_ref=None,
     )
-    theory.num_flavs_init = 4
-    theory.intrinsic_flavors = [4, 5]
-    theory.quark_masses.c.value = 1.0
-    theory.quark_masses.b.value = 4.75
-    theory.quark_masses.t.value = 173.0
+    theory.heavy.num_flavs_init = 4
+    theory.heavy.intrinsic_flavors = [4, 5]
+    theory.heavy.masses.c.value = 1.0
+    theory.heavy.masses.b.value = 4.75
+    theory.heavy.masses.t.value = 173.0
     operator.mu0 = float(np.sqrt(2))
-    operator.mu2grid = [10]
-    operator.rotations.xgrid = XGrid(np.linspace(1e-1, 1, 30))
+    operator.mugrid = [(10, 5)]
```

Comment:
Why now are we adding `10` as `mu` while before we were adding `10` as `mu2`? I expected `np.sqrt(10)`

Reply:
Because in principle is irrelevant :)